docs/integrators/deep-history-squad.md

```diff
@@ -35,143 +39,339 @@ In the example above, the key `726573657276650000000a55544b2d326638306539` is de
 
-#### Bootstrapping
+First, you need to decide whether to reconstruct a **complete** or a **partial** history. A _complete_ history provides the deep-history squad the ability to resolve historical account (state) queries **up until the Genesis**. A _partial_ history, instead, allows it to resolve state queries up until **a chosen past epoch** or **between two chosen epochs**.
 
-First, choose an empty folder to serve as the workspace (working directory) of the squad instance - for example, `~/deep-history-workspace`.
+:::note
+The reconstruction flow has to be performed **for each shard, separately**.
+:::
+
+### Reconstruct a complete history
+
+:::note
+Below, the reconstruction is exemplified for **mainnet, shard 0**. The same applies to other networks (devnet, testnet) and shards (including the metachain). Furthermore, the example refers to the latest Protocol release as of January 2024 (Sirius). When reconstructing the history, use the latest release available.
+:::
 
-Afterward, prepare a configuration file called `reconstruction.json`, following the example of [`default.reconstruction.json`](https://github.com/multiversx/mx-chain-deep-history/tree/main), and save it in the chosen workspace. For the fields `oldestArchive` and `newestArchive`, use URLs towards the MultiversX public archive (which are available [on request](https://t.me/MultiversXDevelopers)). The URLs in the example below are mere placeholders.
+First, set up the reconstruction workspace in a folder of your choice (for the chosen shard), as follows:
 
 ```
-// ~/deep-history-workspace/reconstruction.json
-{
-    "networks": {
-        "devnet": {
-            "shards": {
-                "0": {
-                    "oldestArchive": "https://.../shard-0/2022-October-15.tar",
-                    "newestArchive": "https://.../shard-0/2022-October-25.tar"
-                },
-                "1": {
-                    "oldestArchive": "https://.../shard-1/2022-October-15.tar",
-                    "newestArchive": "https://.../shard-1/2022-October-25.tar"
-                },
-                "2": {
-                    "oldestArchive": "https://.../shard-2/2022-October-15.tar",
-                    "newestArchive": "https://.../shard-2/2022-October-25.tar"
-                },
-                "metachain": {
-                    "oldestArchive": "https://.../shard-metachain/2022-October-15.tar",
-                    "newestArchive": "https://.../shard-metachain/2022-October-25.tar"
-                }
-            }
-        }
-    }
-}
+# In most cases, this should be the latest tag:
+# https://github.com/multiversx/mx-chain-mainnet-config/releases/latest
+export CONFIG_TAG=v1.6.13.0
+
+# Clone mainnet config into a folder named "config"
+git clone https://github.com/multiversx/mx-chain-mainnet-config --branch=$CONFIG_TAG --single-branch --depth=1 config
+
+# In most cases, this should be the tag found in the file "config/binaryVersion":
+export BINARY_TAG=v1.6.13
```

Comment:
maybe get the compatible binary from the `binaryVersion` file after getting the configs?

Reply:
Fixed.